apps/omg_watcher/lib/web/views/transaction.ex

```diff
@@ -23,6 +23,23 @@ defmodule OMG.Watcher.Web.View.Transaction do
   alias OMG.Watcher.Web.Serializer
 
   def render("transaction.json", %{transaction: transaction}) do
+    transaction
+    |> serialize()
+    |> Serializer.Response.serialize(:success)
+  end
+
+  def render("transactions.json", %{transactions: transactions}) do
+    transactions
+    |> Enum.map(&serialize(&1))
+    |> Serializer.Response.serialize(:success)
+  end
+
+  def render("transaction_encode.json", %{transaction: transaction}) do
+    OMG.API.State.Transaction.encode(transaction)
+    |> Serializer.Response.serialize(:success)
+  end
+
+  defp serialize(transaction) do
```

Comment:
`serialize` for the name seems improper here... and `render` is by convention taken... hmm
`render_transaction` :question:

Reply:
I think `serialize ` is a proper name because the function is responsible for preparing the `transaction` object like `Serializer.Response.serialize`is preparing.